From d242c15eb961fbdee69c0a0989cd100d174a25db Mon Sep 17 00:00:00 2001
From: Jonathan Liu <net147@gmail.com>
Date: Wed, 15 Oct 2014 17:01:06 +1100
Subject: [PATCH 6/6] Include QMAKE_CC when extracting C compiler flags

This fixes cross-compile settings detection for cross-compilation
environments where the C compiler cross-compilation flags are set in
CC/QMAKE_CC instead of CFLAGS/QMAKE_CFLAGS (e.g. OpenEmbedded).

Upstream-Status: Backport

Change-Id: I0a7b0f6c47a00775ec18073a16359c8351363285
Reviewed-by: Andras Becsi <andras.becsi@digia.com>
Signed-off-by: Jonathan Liu <net147@gmail.com>
---
 tools/qmake/mkspecs/features/functions.prf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/qmake/mkspecs/features/functions.prf b/tools/qmake/mkspecs/features/functions.prf
index 78b1190..37dd615 100644
--- a/tools/qmake/mkspecs/features/functions.prf
+++ b/tools/qmake/mkspecs/features/functions.prf
@@ -14,7 +14,7 @@ defineReplace(getConfigDir) {
 }
 
 defineReplace(extractCFlag) {
-    CFLAGS = $$QMAKE_CFLAGS
+    CFLAGS = $$QMAKE_CC $$QMAKE_CFLAGS
     !isEmpty(ANDROID_TARGET_CFLAGS): CFLAGS = $$ANDROID_TARGET_CFLAGS
     OPTION = $$find(CFLAGS, $$1)
     OPTION = $$split(OPTION, =)
-- 
2.1.1

